lib/brady.ex

```diff
@@ -11,11 +11,14 @@ defmodule Brady do
 
   """
   @spec body_class(%Plug.Conn{}) :: String.t
-  def body_class(conn) do
+  def body_class(conn = %Plug.Conn{private: %{ phoenix_controller: _ } }) do
```

Comment:
Minor style issue: I think the credo guides don't use a space between the `%{` and the key name